### PR TITLE
feat(compute): Add example to create a single-project reservation

### PR DESCRIPTION
**Pull request by davide-zanon**
Adding file to create single-project reservations by specifying properties directly.

## Description

Fixes #<ISSUE-NUMBER>

Note: If you are not associated with Google, open an issue for discussion before submitting a pull request.

## Checklist

**Readiness**

- [x] Yes, **merge** this PR after it is approved
- [ ] No, don't **merge** this PR after it is approved

**Style**

- [ ] My sample follows the rules described for Terraform in the [Effective Samples style guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [ ] My sample follows the other requirements and best practices in the [Contributing
guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#other-requirements-and-best-practices)

**Testing**

- [ ] I have performed tests described in the [Contributing guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md):

   - [ ] **[Tests](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#set-up-the-test-environment)** pass: `terraform apply`
   - [ ] **[Lint](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#linting-and-formatting)** pass: `terraform fmt` check

**Intended location**

- [ ] Yes, this sample will be (or already is) included on cloud.google.com
      Location(s):

- [ ] No, this sample won't be included on cloud.google.com
      Reason:

**API enablement**

- [ ] If the sample needs an API enabled to pass testing, I have added the service to the [Test setup file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/test/setup/main.tf)

**Review**

- [ ] If this sample adds a new directory, I have added codeowners to the [CODEOWNERS file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/.github/CODEOWNERS)

### CODE REVIEW

**Comment by msampathkumar** [comment]
All tests are passing. Nice work.

Overall, PR LGTM but please fix the conventionalcommit message error.

**Review by msampathkumar — APPROVED** [review]

**Review by gericdong — APPROVED** [review]
LGTM

File: compute/reservation/create_local/main.tf [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START compute_reservation_create_local_reservation]
+
+resource "google_compute_reservation" "deafult" {
```
Comment (rogerthatdev) [inline compute/reservation/create_local/main.tf:19]: Fix typo in `default`

**Review by rogerthatdev — CHANGES_REQUESTED** [review]

**Comment by msampathkumar** [comment]
I fixed the spell-check issues. Approvals are done.

Requester did not request to Merge PR on approval. So, marking this PR as `Waiting response`

**Review by rogerthatdev — APPROVED** [review]

**Pull request merged**